Directory.Build.props

```diff
@@ -22,7 +22,6 @@
   <PropertyGroup Condition="'$(IsTestProject)' == 'true'">
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <GenerateProgramFile>false</GenerateProgramFile>
-    <TestRunnerAdditionalArguments>-parallel none</TestRunnerAdditionalArguments>
```

Comment:
why this change? There are multiple level of parallelism in SDK. The existing test run is fully paralleled. At one point, we have to turn of certain level of parallelism to avoid creating too many process at the same time. If you see this is important, could you run some measurements?

Reply:
I was looking at why my tests weren't running in parallel, and this turned out to be the culprit. It looks like this was changed in master so I wanted to see if if was equally safe to do so in this branch: https://github.com/dotnet/sdk/blob/master/Directory.Build.props#L25